src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private final Storage storage;
+
+    public ReportGeneratorImpl(Storage storage) {
+        this.storage = storage;
+    }
+
+    @Override
+    public String getReport() {
+        StringBuilder report = new StringBuilder("fruit,quantity\n");

```

Comment:
It's better to use System.lineSeparator() instead of '\n' for the newline. It makes your program more portable as it will use the correct newline character(s) for the current platform.